api/http/handler/kubernetes/cluster_role_bindings.go

```diff
@@ -43,3 +45,36 @@ func (handler *Handler) getAllKubernetesClusterRoleBindings(w http.ResponseWrite
 
 	return response.JSON(w, clusterrolebindings)
 }
+
+// @id DeleteClusterRoleBindings
```

Comment:
Would you be able to refer to [services](https://github.com/portainer/portainer/blob/develop/api/http/handler/kubernetes/services.go#L200) deletion and ensure the docs are consistent? For instance, the deletion should return `204`, should check for IsUnauthorized, IsForbidden, and IsNotFound. Also, please separate error log vs HTTP return error!

Reply:
updated